float64 x
float64 y
float64 z
string ClassName
float32 probability
